sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java

```diff
@@ -144,7 +231,7 @@
      * @return A {@link PollerFlux} object.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<PollResult<Void>, Void> beginPurchasePhoneNumbers(String searchId) {
+    public PollerFlux<Void, Void> beginPurchasePhoneNumbers(String searchId) {
         Objects.requireNonNull(searchId, "'searchId' can not be null.");
         return null;
```

Comment:
Is this API required? It doesn't seem to be doing anything. Same for the next two methods.

Reply:
@srnagar Sorry for the confusion here, I wanted to just work on one (searchAvailablePhoneNumbers) and get feedback before continuing with the others. I will be working on these methods now that I know I'm on the right track.